core/primitives-core/src/config.rs

```diff
@@ -223,6 +223,14 @@ pub struct ExtCostsConfig {
     /// Cost of getting sha256 per byte
     pub keccak512_byte: Gas,
 
+    /// Cost of getting ripemd160 base
+    pub ripemd160_base: Gas,
+    /// Cost of getting ripemd160 per message block
+    pub ripemd160_block: Gas,
+
+    /// Cost of calling ecrecover
+    pub ecrecover_base: Gas,
```

Comment:
Shouldn't these be behind a feature flag?

Reply:
That's correct, they should be behind feature flag.

Reply:
Feature flag introduced in https://github.com/near/nearcore/pull/4380/commits/3f08a35fa65f1a24be2fb46f6d72ca1b5c587a3b

Reply:
For some reason this was reverted in https://github.com/near/nearcore/commit/ac53dcbafcc5c0f405eeaf0dccfcdbf5ba19161a#diff-0337a7fe1272b09cad7bda003159822a95e0609b9716fa987b78dacfaa3cd181

Reply:
Was reverted as I was asked to revert this change in this library. See: https://github.com/near/nearcore/pull/4380#discussion_r658889331

This has since be re-reverted. Please code owners do not suggest changes that do not actually help progress. I had assumed that change would've helped the params estimator given conversation context.

Revert: https://github.com/near/nearcore/pull/4380/commits/f25fe99e6421e6131291bd5f1bc4ea1567d13d56